.travis.yml

```diff
@@ -67,3 +74,8 @@ notifications:
 branches:
   except:
     - /^0\..*$/
+
+addons:
```

Comment:
This definition of `addons` nullifies the previous definition of `addons`. You probably mean:

```yaml
addons:
  apt:
    packages:
    - ldap-utils
    - slapd
  hosts:
    - ldap-master
    - ldap-slave
```